src/Api/ZonesApi.php

```diff
@@ -0,0 +1,206 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Api;
+
+use SandwaveIo\RealtimeRegister\Domain\DomainZoneCollection;
+use SandwaveIo\RealtimeRegister\Domain\DomainZoneRecordCollection;
+use SandwaveIo\RealtimeRegister\Domain\DomainZoneStatistics;
+use SandwaveIo\RealtimeRegister\Domain\DomainZoneUpdate;
+use SandwaveIo\RealtimeRegister\Domain\Zone;
+
+final class ZonesApi extends AbstractApi
+{
+    /**
+     * @see https://dm.realtimeregister.com/docs/api/dns/zones/get
+     */
+    public function get(int $zoneId): Zone
+    {
+        $response = $this->client->get("v2/dns/zones/{$zoneId}");
+        return Zone::fromArray($response->json());
+    }
+
+    /** @see https://dm.realtimeregister.com/docs/api/dns/zones/list */
+    public function list(
+        ?int $limit = null,
+        ?int $offset = null,
+        ?array $includedFields = null,
+    ): DomainZoneCollection {
+        $query = [];
+        if (! is_null($limit)) {
+            $query['limit'] = $limit;
+        }
+        if (! is_null($offset)) {
+            $query['offset'] = $offset;
+        }
+
+        if (! is_null($includedFields)) {
+            $query['fields'] = implode(',', $includedFields);
+        }
+
+        $response = $this->client->get('v2/dns/zones', $query);
+        return DomainZoneCollection::fromArray($response->json());
+    }
+
+    /** @see https://dm.realtimeregister.com/docs/api/dns/zones/create */
+    public function create(
+        string $name,
+        ?string $service = null,
```

Comment:
Do we want to restrict this to DomainZoneServiceEnum?

Reply:
I would suggest not creating an enum for service. Current possible values are BASIC and PREMIUM, but I can easily see that types will be added to this list, whenever marketing comes up with a new product.
Not that that would be a huge problem, but that would mean with every type being added, that this client needs to be updated and in the meantime the client can't be used for these new types.